api/src/main/java/io/opencensus/common/ToDoubleFunction.java

```diff
@@ -17,7 +17,7 @@
 package io.opencensus.common;
 
 /**
- * Represents a function that produces a double-valued result. See {@link
+ * Represents a function that produces a double-valued result. See {@code
```

Comment:
I had to replace the link to the metrics class with the class name in code font because the class is no longer accessible from the API library.  I think it will be easy to just revert this commit because git handles moved files well.